src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -348,14 +348,14 @@ def get_api_pvs():
     return _output_parser(stdout, fields)
 
 
-def get_lv_from_argument(argument):
+def get_lv_from_argument(argument, lvs):
```

Comment:
is it required to always pass `lvs` here? Do you foresee any situations where it might not need it?

Reply:
Oh, no. This should be initialized to `None` by default like rest of the methods that take `lvs` as an argument.

Reply:
Fixed.